lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -213,7 +219,11 @@ def queue_job(self, job_wrapper):
         self.monitor_queue.put(ajs)
 
     def __has_guest_ports(self, job_wrapper):
-        return bool(job_wrapper.guest_ports)
```

Comment:
At some points (like when interactive tools are stopped) the notion of guest_ports gets stripped from the job and we need to add other ways of checking whether this is an interactive job or not. Maybe we should rename this method.